lms/templates/main.html

```diff
@@ -160,6 +160,10 @@
     </script>
 % endif
 
+% if configuration_helpers.get_value('GOOGLE_TAG_MANAGER_SNIPPET'):
+    ${configuration_helpers.get_value('GOOGLE_TAG_MANAGER_SNIPPET') | n}
```

Comment:
What does the "n" filter mean? @ivanvgh

Reply:
@Squirrel18, here it is the documentation. https://docs.makotemplates.org/en/latest/filtering.html#turning-off-filtering-with-the-n-filter